README.md

```diff
@@ -117,6 +117,7 @@ Useful articles and learning resources.
 - [Free Code Camp](https://www.freecodecamp.org) - Software development related posts
 - [CSS Tricks](https://css-tricks.com) - CSS related posts
 - [DEV.to](https://dev.to) - The All in One Blogs Portal for Developers
+- [Stephen Jude's Blog](http://stephenjude.me/) - Everything about PHP, Laravel and
```

Comment:
Thanks for this amazing PR!

Can confirm that the description is complete?

Reply:
No Chief @BolajiAyodeji it isn't. let me update it

Reply:
Updated @BolajiAyodeji